src/udev/udevd.h

```diff
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: GPL-2.0-or-later */
 #pragma once
 
-int run_udevd(int argc, char *argv[]);
+#include <stdbool.h>
```

Comment:
why is this include needed here?

Reply:
Ouch. I mixed in something wrong way. Resolved.